supplier_calendar/README.rst

```diff
@@ -0,0 +1,115 @@
+=================
+Supplier Calendar
+=================
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:c4f6b16596003384effca4ac9aff2db58bf56e51a47b17c6703801ac4b3edf5e
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fpurchase--workflow-lightgray.png?logo=github
+    :target: https://github.com/OCA/purchase-workflow/tree/18.0/supplier_calendar
+    :alt: OCA/purchase-workflow
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/purchase-workflow-18-0/purchase-workflow-18-0-supplier_calendar
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/builds?repo=OCA/purchase-workflow&target_branch=18.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+This module adds a Calendar to the ResPartner model. This calendar can
+then used as the basis of the proper computation of start/end dates
+based on the delivery lead time of the supplier in this and other
+modules.
+
+In this module, the calendar is considered in the computation of the
+schedules date of a stock picking and in the order date of a purchase
+order.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+-  Go to *Settings* and activate the developer mode.
+-  Go to *Settings > Technical > Resource > Working Schedules* and
+   define your resource calendar.
+-  Go to *Contacts > Sales&Purchases > Purchase > Delay Calendar Type*
+   and assign Supplier Calendar and in *Factory Closing Days* assign the
+   Resource Calendar desired.
```

Comment:
Could you correct this? it should be "Factory Calendar" to much the view:

![image](https://github.com/user-attachments/assets/b781ba78-e22a-4b18-9758-cf17f5aa12bf)

Reply:
Changed, Thanks!